### PR TITLE
PEPs 2, 4, 411: updates to how the stdlib is managed

### PR DESCRIPTION
**Pull request by brettcannon**
- New top-level modules require a PEP.
- Deprecating a top-level module requires a PEP.
- Provisional modules didn't work out.

### CODE REVIEW

**Comment by brettcannon** [comment]
This is draft until the rest of the SC has a chance to check my work.

File: pep-0002.txt
Comment (AA-Turner) [inline pep-0002.txt:48]: With the same change to footnote 1:
```suggestion
standard library, the overall process, etc. are covered in `the developer's
guide`_.
```

File: pep-0002.txt
Comment (AA-Turner) [inline pep-0002.txt:59]: Suggested to delete from "and not force" -- this seems just a repetition of the earlier idea, but in stronger language.
```suggestion
to keep their external code in sync as appropriate.
```

File: pep-0411.txt
Comment (AA-Turner) [inline pep-0411.txt:16]: Fix markup:
```suggestion
   This PEP has been marked as *obsolete*. A decade after this PEP
```

File: pep-0411.txt
Comment (AA-Turner) [inline pep-0411.txt:7]: We don't have an Obsolete header. You could perhaps also add `Superseded-By: 2`.
```suggestion
Status: Superseded
```

**Review by AA-Turner — COMMENTED** [review]
The copyright and emacs stanzas on the 2, 4, and 411 might all need updating / removing respectively.

A

**Comment by brettcannon** [comment]
I'm fine with deleting the Emacs stuff, but these PEPs are so old I don't know if we can update the copyrights as I don't know if all the contributors signed the CLA.

**Review by CAM-Gerlach — COMMENTED** [review]
Since this is nearly a complete re-write of a PEP 2 and PEP 4, marks a previously Active PEP as Superseded and a previously Superseded PEP as Active, and will presumably need to go through the PEP discussion and approval process anyway given the magnitude of the change, wouldn't it be cleaner to just instead put this in a new single PEP, e.g. `Adding and Removing Standard Library Modules`, that marks the older ones as superseded?

This would:

* Put the new process for adding and removing stdlib modules, which is short, to the point and motivated by a common basic rationale, in one place, making things simpler for readers and reviewers.
* Preserve the existing accepted PEPs and their normative and informational contents as historical documents
* Prevent any confusion over a previously superseded PEP being updated and superseding an active PEP that previously superseded it
* Avoid the whole copyright/license issue and any confusion about authorship

@python/pep-editors thoughts?

File: pep-0002.txt
Comment (CAM-Gerlach) [inline pep-0002.txt:63]: Sorry, forgot to include this in my initial review—

I can't quite reach it, but this can simply be removed after using a proper inline link instead of a footnote for the one reference within.

File: pep-0002.txt
Comment (gvanrossum) [inline pep-0002.txt:18]: If this PEP was superseded, leave it as a historical document, and create a new one?

File: pep-0002.txt
Comment (gvanrossum) [inline pep-0002.txt:23]:
```suggestion
Due to the visibility and importance of the standard library, it must
```

File: pep-0004.txt
Comment (gvanrossum) [inline pep-0004.txt:80]: Again, I think it's better to keep this PEP around as a historical document and just declare it to be superseded by 387.

File: pep-0002.txt
Comment (CAM-Gerlach) [reply pep-0002.txt:18]: Indeed, very much agreed, see [my comment](https://github.com/python/peps/pull/2651#pullrequestreview-1019030218) to that effect.

File: pep-0004.txt
Comment (CAM-Gerlach) [reply pep-0004.txt:80]: Very much agreed as well

**Comment by brettcannon** [comment]
I talked it over with the SC and we are good with changing the PEPs in-place. When I have a chance, I will look at the review comments.

**Review by JelleZijlstra — APPROVED** [review]

File: pep-0002.txt
Comment (brettcannon) [reply pep-0002.txt:49]: Because I wasn't trying to rewrite the entire PEP from scratch, and so changing its style was not a priority to me.

File: pep-0002.txt
Comment (brettcannon) [reply pep-0002.txt:58]: That's Jason's decision and it doesn't contradict this statement. The point that other core devs can view the stdlib copy as canonical and not concern themselves w/ code outside of the repo still holds.

**Pull request merged**
